README.md

```
@@ -1,12 +1,14 @@
 # Transcript Seeker
 
+<p align="center"><a href="https://discord.com/invite/dsvFgDTr6c"><img height="60px" src="https://user-images.githubusercontent.com/31022056/158916278-4504b838-7ecb-4ab9-a900-7dc002aade78.png" alt="Join our Discord!"></a></p>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: p

(MD033, no-inline-html)